<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>217</width>
      <height>234</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(81, 0, 244);
font: 75 20pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Set another time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="ChangeButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Change Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(81, 0, 244);
font: 75 20pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>OR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="CancleButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Cancel The Plan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
